src/Router.jsx

```diff
@@ -0,0 +1,53 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+
+import NotFound from './pages/NotFound';
```

Comment:
It could be nice to at least group the import by a shared topic. For example:
```
// Pages
...
// Account pages
...
// Books pages
...
// Cart pages
...
// Auth pages
...
// Layouts
```
This is up to you but it could improve code readability.